src/front/styles/Header.css

```diff
@@ -0,0 +1,29 @@
+header {
+  color: white;
+  background: #3da2daff;
+  background-image: url("https://github.com/nannyhub/NannysHub/blob/landingpage/src/front/img/nan.jpg");
+  min-height: 50vh;
+  padding-top: 20vh;
+  text-align: center;
+  position: relative;
+  z-index: 1;
+  overflow: hidden;
+  background-position: center; /* Center the image */
+  background-repeat: no-repeat;
+  background-size: cover; /* Resize the background image to cover the entire container */
+}
+
+header h1 {
```

Comment:
Oooh... This is a *very* dangerous CSS rule to apply... You're telling that EVERY `<h1>` that comes after the `<header>` will inherit these properties.

Maybe it is exactly what you meant, but... I think it's better to create a class for cases like this, because in more complex pages you could have a literal chaos over the CSS styles because of this hahahah